velox/connectors/Connector.h

```diff
@@ -171,20 +179,30 @@ class ConnectorQueryCtx {
  public:
   ConnectorQueryCtx(
       memory::MemoryPool* pool,
-      Config* config,
+      const core::QueryConfig& queryConfig,
+      Config* connectorConfig,
       ExpressionEvaluator* expressionEvaluator,
       memory::MappedMemory* mappedMemory,
-      const std::string& scanId)
+      const std::string& taskId,
+      const std::string& planNodeId,
+      int driverId)
       : pool_(pool),
-        config_(config),
+        queryConfig_(&queryConfig),
+        config_(connectorConfig),
         expressionEvaluator_(expressionEvaluator),
         mappedMemory_(mappedMemory),
-        scanId_(scanId) {}
+        scanId_(fmt::format("{}.{}", taskId, planNodeId)),
```

Comment:
The comment for scanId getter needs updating.

Reply:
Can I ask what the comment should look like? I see the scanId is used by HiveDataSource and here https://fburl.com/qr41uqit.

Reply:
The comment says "This is typically a query id plus the scan's PlanNodeId." This was fine when scan ID was provided externally. Now, scan ID is computed directly, hence, we know exactly what it is. Therefore, the word "typically" does not apply anymore. I'd change this sentence to

`This is typically a combination of task id and the scan's PlanNodeId.`